website/notifications/utils.py

```diff
@@ -236,9 +236,11 @@ def format_data(user, node_ids):
                              if getattr(subscription, 'event_name') in node_sub_available]
             for subscription in subscriptions:
                 index = node_sub_available.index(getattr(subscription, 'event_name'))
-                children.append(serialize_event(user, subscription=subscription,
+                if not (not node.mailing_enabled and subscription.event_name == 'mailing_list_events'):
```

Comment:
Can't this condition be reduced to `subscription.event_name != 'mailing_list_events'`?